compiler/ETA/CodeGen/Closure.hs

```diff
@@ -61,20 +61,21 @@ lfStandardForm args fvs
 
 lfClass :: Bool -> Int -> Int -> LambdaFormInfo -> Text
 lfClass hasStdLayout _arity fvs (LFThunk {..}) =
-  "eta/runtime/thunk/" <> thunkBase <> thunkExt
+  "eta/" <> thunkBase <> thunkExt
   where thunkBase
-          | isTopLevel lfTopLevelFlag = "CAF"
+          | isTopLevel lfTopLevelFlag = "runtime/thunk/CAF"
           | lfUpdatable               = "UpdatableThunk"
-          | otherwise                 = "SingleEntryThunk"
+          | otherwise                 = "runtime/thunk/SingleEntryThunk"
         thunkExt
           | isTopLevel lfTopLevelFlag = mempty
-          | hasStdLayout = T.pack (show fvs)
+          | hasStdLayout = T.pack ((++) (show fvs) "runtime/thunk/")
```

Comment:
@rahulmutt I've tried `| hasStdLayout = T.pack ((++) (show fvs) "runtime/thunk/")` and `| hasStdLayout = T.pack ((++) "runtime/thunk/" (show fvs) )` still I get `Caused by: java.lang.ClassNotFoundException: eta.UpdatableThunk5runtime.thunk.`error. Any idea?

Reply:
Why are you trying to add `runtime/thunk/` instead of removing it above? Have you changes the names of all the `eta/runtime/thunk/*Thunk*` classes to `eta/*Thunk*`?

Reply:
@rahulmutt The classes Thunk, UpdatableThunk are only moved to package eta.
The remaining  *Thunk* classes are present in package package eta/runtime/thunk.
when I make package as eta , I get getting CNFE for java.lang.NoClassDefFoundError: eta/UpdatableThunk2.
when I make package as eta/runtime/thunk/ , I get getting CNFE for java.lang.NoClassDefFoundError: eta/runtime/thunk/UpdatableThunk.
so, I'm trying to find  way to include both eta/UpdatableThunk and eta/runtime/thunk/**.

Reply:
I think it will be easier if you shifted all the `eta/runtime/thunk/**` classes to `eta/*`. That way you won't have to add special cases in the logic.

Reply:
I think it's fine to move up the specialized thunks and functions up to the `eta` package. They will be exposed in the public runtime API for those who need it.